pkg/browsers/console.go

```diff
@@ -304,9 +310,12 @@ func makeDestinationURL(service string, region string) (string, error) {
 
 	dest := prefix + serv + "/home"
 
-	//NOTE here: excluding iam here and possibly others as the region isnt in the uri of the webpage on the console
-	if region != "" || serv != "iam" {
+	//excluding region here if the service is apart of the global service list
+	//uncomplete list of global services
+	notGlobal := globalServiceMap[service] == ""
+	if notGlobal && region != "" {
 		dest = dest + "?region=" + region
+
 	}
```

Comment:
A more 'Go-like' way to do this is using the second return value from a map lookup to check whether it exists.
```
val, ok := dict["foo"]
```

```suggestion
	_, global := globalServiceMap[service]
	hasRegion := region != ""
	
	if !global && hasRegion {
		dest = dest + "?region=" + region
	}
```